algorithms/searching.md

```diff
@@ -41,17 +35,17 @@ Think about a shuffled deck of cards. Let's say you're looking for the 6 of hear
 
 If you have a phone book, and you're looking for Jessica Smith, do you start on page 1 and read every name until you find her? Probably not. By starting in the middle, flipping larger chunks of pages until you get to the Ss, then eventually finding Smith, then finding Jessica eliminates a TON of time. This is the idea behind a binary search.
 
-All of these algorithms are pretty common, and have standard implementations in most programming languages. But understanding how they work and when to use one over another will help you write better software.
+All of these algorithms are pretty common and have standard implementations in most programming languages. But understanding how they work and when to use one over another will help you write better software.
 
 Watch the video to learn more details about linear and binary search algorithms!
 
-Slides: https://docs.google.com/presentation/d/1x8xO_URwZVoB4JxTNN90z0sHSxoYEq0INOH_IIi8SLE/edit#slide=id.p
+Slides: [Read this slides, It is also available in the materials section](https://docs.google.com/presentation/d/1x8xO_URwZVoB4JxTNN90z0sHSxoYEq0INOH_IIi8SLE/edit#slide=id.p)
```

Comment:
Using the same labels for the slides and video that are used in the materials section would be a better choice than these sentences.